test/unit/selendroid-specs.js

```diff
@@ -3,9 +3,9 @@
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import 'mochawait';
-import { SelendroidServer } from '../lib/selendroid';
+import { SelendroidServer } from '../../lib/selendroid';
 import { fs } from 'appium-support';
-import { SE_APK_PATH, SE_MANIFEST_PATH } from '../lib/setup';
+import { SE_APK_PATH, SE_MANIFEST_PATH } from '../../lib/setup';
```

Comment:
Yeah. After merging this, I'll just go ahead and update @sebv's PR and fix merge conflicts